src/Bridges/bridgeoptimizer.jl

```diff
@@ -239,12 +239,25 @@ end
 function MOI.supports(b::AbstractBridgeOptimizer,
                       attr::MOI.AbstractConstraintAttribute,
                       IndexType::Type{<:MOI.Index})
-    return MOI.supports(b.model, attr, IndexType)
+    if is_bridged(b, IndexType)
+        return MOI.supports(b.bridged, attr, IndexType)
+    else
+        return MOI.supports(b.model, attr, IndexType)
+    end
 end
+
 function MOI.set(b::AbstractBridgeOptimizer,
                  attr::MOI.AbstractConstraintAttribute,
                  index::MOI.Index, value)
-    return MOI.set(b.model, attr, index, value)
+    if is_bridged(b, typeof(index))
+        if MOI.is_set_by_optimize(attr)
```

Comment:
This should be `if !(attr isa MOI.ConstraintName)` (also modify in `MOI.get`)

Reply:
`MOI.get` throws an error when I try to retrieve attributes `ConstraintFunction` and `ConstraintSet`. Following code works : 
```julia
    if is_bridged(b, typeof(ci))
        if attr isa MOI.ConstraintName || attr isa MOI.ConstraintFunction ||
                attr isa MOI.ConstraintSet
            MOI.get(b.bridged, attr, ci)
        else
            MOI.get(b, attr, bridge(b, ci))
        end
    else #...
```

Reply:
Error message I got when I run the tests
> ArgumentError: Constraint bridge of type `MathOptInterface.Bridges.SplitIntervalBridge{Float64,MathOptInterface.ScalarAffineFunction{Float64}}` does not support accessing the attribute `MathOptInterface.ConstraintSet()`.